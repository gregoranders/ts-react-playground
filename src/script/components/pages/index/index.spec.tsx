import React, { mount, create } from '@app/testUtils';

import * as TestSubject from '@pages/index';

describe(`${TestSubject.IndexPage.displayName}`, () => {
  it('exports default', () => {
    expect(TestSubject.default).toBeDefined();
  });

  const TestComponent = TestSubject.default;

  it('snapshot', (): void => {
    const snapshot = create(<TestComponent />);
    snapshot.toJSON();
    expect(snapshot).toMatchSnapshot();
  });

  it('mount', () => {
    const testSubject = mount(<TestComponent />);
    expect(testSubject.text()).toBe('IndexPage');
  });
});
