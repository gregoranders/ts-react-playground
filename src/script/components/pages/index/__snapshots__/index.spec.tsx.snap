// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage snapshot 1`] = `
<h2>
  IndexPage
</h2>
`;
